src/applications/ask-va/containers/ResponseInboxPage.jsx

```diff
@@ -218,17 +239,9 @@ const ResponseInboxPage = ({ loggedIn }) => {
                   ],
                 });
               }}
-              secondary
-              text={RESPONSE_PAGE.UPLOAD_BTN}
+              uswds
```

Comment:
The `uswds` prop can be removed.

Reply:
Noted for the follow up ticket for Dashboard